docs/content-development/guide.md

```diff
@@ -166,23 +166,1258 @@ spec:
       {
```

Comment:
I think it makes sense to keep in sync. When I first read the documentation, cloned the repo, and open the Mattermost installation implementation, it would make me confused if the file there would be different from the one created in the docs. Or at least, we should mention that for instance: "For simplicity, we modified the Mattermost installation manifest to contains only the most valuable parts. 

I mentioned the generator and create the issue for it: https://github.com/capactio/website/issues/115

Reply:
Due to the fact that there was falling CI jobs due to too long document, I decided to keep only `host`. I tested it locally and the manifest works. :)